docs/8-the-supporting-acts/7-image-security.md

```diff
@@ -0,0 +1,140 @@
+# Image Security (StackRox)
+
+> We use some ready container images to build our modelcars 🚗. We use them as base and add our model artifacts on top of them. We need to protect our containers from errors and vulnerabilities in public registry images, or outdated packages and libraries. Image security step in our pipeline helps us to discover them before moving images to production.
+
+
+## Set Up StackRox Access
+
+StackRox (Advanced Cluster Security, or ACS) is deployed at the cluster level and can monitor multiple clusters. In this environment, the ACS / StackRox operator has already been deployed and configured for you.
+
+1. Connect to the ACS WebUI:
+
+    ```bash
+    # get web url
+    echo https://$(oc -n rhacs-operator get route central --template='{{ .spec.host }}')
+    ```
+
+    Using the **admin** username:
+
+    ```bash
+    # get password to go with the "admin" username:
+    echo $(oc -n rhacs-operator get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}')
```

Comment:
We have access to this as regular users?

Reply:
ah don't mind ACS ones